pagerduty/resource_pagerduty_service_test.go

```diff
@@ -242,6 +242,83 @@ func TestAccPagerDutyService_AlertContentGrouping(t *testing.T) {
 	})
 }
 
+func TestAccPagerDutyService_AutoPauseNotificationsParameters(t *testing.T) {
+	username := fmt.Sprintf("tf-%s", acctest.RandString(5))
+	email := fmt.Sprintf("%s@foo.test", username)
+	escalationPolicy := fmt.Sprintf("tf-%s", acctest.RandString(5))
+	service := fmt.Sprintf("tf-%s", acctest.RandString(5))
+
+	resource.Test(t, resource.TestCase{
+		// PreCheck:     func() { testAccPreCheck(t); testAccPreCheckPagerDutyAbility(t, "XXX") }, // FIXME: not sure which ability this feature is depending on https://developer.pagerduty.com/api-reference/9def78ed82b74-list-abilities
```

Comment:
I am not sure which ability this feature is depending on https://developer.pagerduty.com/api-reference/9def78ed82b74-list-abilities